src/examples/forms/inputs/Validation.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { Field, Label, Input, Message } from '@zendeskgarden/react-forms';
+
+const StyledRow = styled(Row)`
+  @media (max-width: ${props => props.theme.breakpoints.md}) {
+    display: flex;
+    flex-direction: column;
```

Comment:
A `Row` is already `display: flex` and `flex-direction` should not be used with Garden's grid. Use props on the `Col`, instead. i.e. `<Col sm>` to stack at the small breakpoint.

So, in order to get vertical spacing, this becomes (replaces the 2nd and 3rd columns):

```js
const StyledCol = styled(Col)`
  @media (max-width: ${p => p.theme.breakpoints.sm}) {
    margin-top: ${p => p.theme.space.sm};
  }
`;
```

...but at this point, it's questionable for me whether this is a pattern we want to push through the examples. If so, let's tighten-up throughout. If not, then revert to using `<Col sm>` when columns need to stack on mobile.